lib/ansible/module_utils/common/yaml.py

```diff
@@ -0,0 +1,32 @@
+# (c) 2020 Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+This file provides ease of use shortcuts for loading and dumping YAML,
+preferring the YAML compiled C extensions to reduce duplicated code.
+"""
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from functools import partial
+
+import yaml
```

Comment:
Being a module_util, this will need a `try/except ImportError` block in case `yaml` isn't available.